src/felles-komponenter/EkspanderbartTekstomrade.tsx

```diff
@@ -0,0 +1,45 @@
+import { NedChevron, OppChevron } from 'nav-frontend-chevron';
+import React, { useState } from 'react';
+
+import styles from './EkspanderbartTekstomrade.module.less';
+
+const MoreOrLess = ({ text, maxCharacters }: { text: string; maxCharacters: number }) => {
+    const [hasLongText, setHasLongText] = useState(true);
+    const toggleMoreOrLess = () => {
+        setHasLongText(!hasLongText);
+    };
+    if (text.length > maxCharacters) {
+        return (
+            <p>
+                {hasLongText ? text.slice(0, maxCharacters) + ' ... ' : text + ' '}
+                <span className={styles.nobreak}>
+                    <span className={styles.color}>
```

Comment:
Er det mulig å slå disse to sammen? Er mange spans inni hverandre her og det kan bli litt uoversiktlig

Reply:
Ønsket å ha nobreak mellom ellipse, label og chevron, men bare farge på de to siste. Husker ikke om chevron-komponenten tok className, kan fikse om så.